frontend/integration-tests/tests/filter.scenario.ts

```diff
@@ -72,7 +72,7 @@ describe('Filtering', () => {
   });
 
   it('filters Deployment from other namespace', async () => {
-    await browser.get(`${appHost}/k8s/ns/kube-system/deployments`);
+    await browser.get(`${appHost}/k8s/ns/openshift-apiserver/deployments`);
```

Comment:
This is a better change, but won't work in native k8s, only OpenShift.

It looks like we don't have a test for filtering in a namespace-scoped list page. So I'd just make this...

```ts
  it('filters Deployment from namespace-scoped list', async () => {
    await browser.get(`${appHost}/k8s/ns/${testName}/deployments`);
```

Reply:
So we need such a namespace that has some other deployment but not the one we created. So that the Filter is shown but when searched for our deployment it gives us the `No deployments found` screen. Thus I will remove the part where we input to input since we don't have one and restore to original namespace.

Reply:
There's no need for the test if we don't input anything. I would be OK just removing this test honestly. Filtering from the list page is well-covered by the crud scenario.